.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Tests
+on: push
```

Comment:
This should be on pull and targeting the `main` branch

Reply:
Since the compute for the CI is free I would not see a reason not to have the CI trigger for all pushes to all branches, but I'm not precious about this